src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
+          project_id: s.project.id,
+          timeToFinish: (12 - moment.utc().diff(s.assigned_at, 'hours')),
```

Comment:
This `checkAssigned` function needs to be refactored. The `assigned` array doesn't seem to be very useful as is. I used to use it for debugging and now it doesn't look like it's used for anything. I think it would be useful for this scenario if it simply held the currently assigned submissions, as they are returned from the API.